test/L1ERC20Bridge.t.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {Test} from "forge-std/Test.sol";
+
+import {IERC20Mint} from "src/interfaces/IERC20Mint.sol";
+import {FakeERC20} from "src/FakeERC20.sol";
+import {L1ERC20Bridge} from "src/L1ERC20Bridge.sol";
+import {Constants} from "test/Constants.sol";
+
+contract L1ERC20BridgeTest is Test, Constants {
+  IERC20Mint erc20;
+
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl("avalanche_fuji"));
+    erc20 = new FakeERC20("Hello", "WRLD");
+  }
+}
+
+contract Deposit is L1ERC20BridgeTest {
+  function testFork_CorrectlyDepositTokens() public {
+    L1ERC20Bridge bridge = new L1ERC20Bridge(address(erc20), wormholeCoreFuji);
+    bridge.initialize(0xBaA85b5C4c74f53c46872acfF2750f512bcBEC43);
+
+    erc20.approve(address(bridge), 100_000);
+    erc20.mint(address(this), 100_000);
+    vm.deal(address(this), 1 ether);
+
+    bridge.deposit(address(this), 100_000);
```

Comment:
*RR* There are no test assertions here. Do we want to check anything? Or are these tests meant as placeholders for now?

Reply:
I was planning to handle tests in a followup pr. These are meant to be check and make sure nothing is reverting before we test with an end to end test.

Reply:
I can add tests to this if it would be easier to review in this pr

Reply:
Got! No problem, if that's your intent I'm fine with it being more broken up. Just wanted to sanity check.